### PR TITLE
Clarify the ruby version

### PR DESCRIPTION
**Pull request by brunoocasali**
Addressing this comment: https://github.com/meilisearch/meilisearch-ruby/pull/533#issuecomment-2099072353

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ Say goodbye to server deployment and manual updates with [Meilisearch Cloud](htt
 
 ## 🔧 Installation
 
-This package requires Ruby version 3.0.0 or later.
+We officially support any version of Ruby that is still being officially maintained, but in any case, you can use any version above Ruby 3.0.
```
Comment (ellnix) [inline README.md:56]:
```suggestion
We officially support any version of Ruby that is still receiving at least [security maintenance](https://www.ruby-lang.org/en/downloads/branches/). You may, however, be fine with any Ruby version above 3.0.
```

**Review by ellnix — CHANGES_REQUESTED** [review]
I like the idea, just a suggestion to make the promise a little bit more precise.

**Review by ellnix — APPROVED** [review]
LGTM.

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**